src/compiler/reverse.jl

```diff
@@ -3,6 +3,18 @@ using IRTools: IR, Variable, Pipe, xcall, var, prewalk, postwalk,
   insertafter!, finish, expand!, prune!, substitute!, substitute,
   block, block!, branch!, return!, stmt, meta
 
+
+# TODO: Temporary, to be removed when ChainRulesCore rrules are required to
```

Comment:
would be good to link to the issue/PR here

Reply:
```suggestion
# TODO: Temporary, to be removed when https://github.com/JuliaDiff/ChainRules.jl/issues/408 is done
# and ChainRulesCore rrules are required to
```